log4j-api-test/src/main/java/org/apache/logging/log4j/test/junit/StatusLoggerExtension.java

```diff
@@ -146,10 +148,16 @@ public void handleException(final ExtensionContext context, final Throwable thro
                         .withThrowable(data.getThrowable())
                         .withLocation(data.getStackTraceElement())
                         .log("{} {}",
-                                DateTimeFormatter.ISO_LOCAL_TIME.format(Instant.ofEpochMilli(data.getTimestamp())),
+                                formatLocalTime(data.getTimestamp()),
                                 data.getMessage());
             });
         }
+
+        private static String formatLocalTime(final long epochMilli) {
+            final Instant instant = Instant.ofEpochMilli(epochMilli);
+            final LocalDateTime localDateTime = LocalDateTime.ofInstant(instant, ZoneId.systemDefault());
+            return localDateTime.format(DateTimeFormatter.ISO_LOCAL_TIME);
+        }
```

Comment:
Turns out this change didn't work, so I fixed it and forgot to push to 2.x directly, so here you go.

Reply:
Somehow I always get the time API wrong. It should be:
```
DateTimeFormatter.ISO_LOCAL_TIME.withZone(ZoneId.systemDefault()).format(Instant.ofEpochMilli(data.getTimestamp())),
```

Reply:
Oh that makes sense; something needs to be converted to a time zone.

Reply:
I pushed this change to both `2.x` and `main` (it was also breaking #1961), so you can rebase.